src/config/theme.js

```diff
@@ -60,6 +63,7 @@ export default {
 	[INPUT]: input,
 	[NUMERIC_STEPPER]: numericStepper,
 	[SELECTBOX]: selectbox,
+	[SELECTBOX_ANCHOR]: selectboxAnchor,
```

Comment:
I believe there's no need in separate theme for anchor defined in context. We should minimize amount of context-themed components.